install/powershellscript/cmd/microsoftgoinstallscript/main.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Meta question: can we omit the microsoft prefix from the script name and from the new package name? In the future we might want to support downloading the upstream toolchain using this same script and just changing a parameter (which will always default to the Microsoft toolchain). If that happens, then having the Microsoft prefix will feel strange.

I guess the Microsoft prefix is there to avoid conflicting with upstream tooling, but I don't think it will ever conflict. Google want provide a powershell script, and even if it did, users installing both could rename our script with whatever name they like.